### DISCUSSION TITLE
facing error on changing speed

### DISCUSSION THREAD

**Discussion by nadia-am** [CoderLine/alphaTab#621] [Q&A] [answered]
as the documentation in [playbackspeed](https://www.alphatab.net/docs/reference/api/playbackspeed/) , but i am getting error: 
`(index):1145 Uncaught TypeError: api.playbackSpeed is not a function`
![image](https://user-images.githubusercontent.com/11175468/121807146-356e0580-cc68-11eb-8b28-b4cbb1fa3a25.png)

what is the solution? where am i wrong?!

**Comment by Danielku15** [comment, marked as answer]
Seems the docs are wrong in this area. playbackSpeed is not a function but a property. You need to assign a value to it like `api.playbackSpeed = 0.5`. I will correct the docs.
